src/rpc/net.cpp

```diff
@@ -29,6 +29,25 @@
 
 #include <univalue.h>
 
+static const std::vector<std::tuple<int, const std::string, const std::string>> CONN_TYPES{
+    std::make_tuple(0, "inbound", "initiated by the peer"),
```

Comment:
I think this is complicated and fragile. This assumes `0` to be `ConnectionType::INBOUND` and if the enum is changed then this brakes.

Reply:
This should be changed neither lightly nor frequently, it at all, and here the changes to be made are in one method only.

Reply:
Removed this. Thanks for the feedback.